pulp_2to3_migration/app/pre_migration.py

```diff
@@ -479,17 +479,20 @@ def pre_migrate_distributor(repo_id, distributors, distributor_migrators, repo=N
                 distributor.pulp2_config = dist_data.config
                 distributor.pulp2_last_updated = last_updated
                 distributor.is_migrated = False
+                distributor.save()
                 dist_migrator = distributor_migrators.get(distributor.pulp2_type_id)
                 needs_new_publication = dist_migrator.needs_new_publication(distributor)
                 needs_new_distribution = dist_migrator.needs_new_distribution(distributor)
                 remove_publication = needs_new_publication and distributor.pulp3_publication
                 remove_distribution = needs_new_distribution and distributor.pulp3_distribution
+
                 if remove_publication:
+                    # TODO check when publication shared by multiple distro
+                    # FIXME when checksum_type is added to the publication
```

Comment:
How does checksum affect this code? (it's definitely affecting `dist_migrator.needs_new_publication` implementation for RPM, I believe there should be a comment there )

Reply:
i left a todo here because right now this code is not executed, once it will be executed we will remove the publication, we need to make sure that we cover properly the case when the publication is shared by multiple distributions.

Reply:
I also have added some code around this case but i have not really properly tested it, so we can whether leave the codo and the code, or commented out code, i am ok both ways

Reply:
@ipanova , I commented the line with FIXME which comments about checksum type.